### DISCUSSION TITLE
distributing/deploy-cli

### DISCUSSION THREAD

**Discussion by giscus** [velopack/velopack.docs#24] [General]
# distributing/deploy-cli

The general process for deploying a Velopack release (download -> pack -> upload) can be greatly simplified by using the download and upload commands which are built into the vpk command line tool.

https://docs.velopack.io/distributing/deploy-cli

**Comment by shunkica** [comment]
"It is tedious to update this file manually, so Velopack CLI provides deployment commands which can deploy assets and update this file automatically for you, as well as apply rentention policies around the number of releases to keep. " This text leads to this page, but there is no description on how to specify the number of releases to keep. Is there any clear way to remove old releases from the json file except manually?

**Reply by caesay** [reply]
Please review the upload cli help for options like '--keepMaxReleases'

**Reply by Borja-DS** [reply]
Is that option available on GitHub upload? Or there is a way of achieving something similar? I can't find it on the docs.

**Reply by caesay** [reply]
Review `vpk upload github -h` for the Github upload command help.
